synapse/models/inet.py

```diff
@@ -1470,10 +1479,10 @@ async def _onSetWhoisText(node, oldv):
                 'doc': 'An object status enumeration.'}),
 
             ('inet:service:account', ('guid', {}), {
+                'template': {'title': 'service account'},
                 'interfaces': (
-                    ('inet:service:subscriber', {'template': {
-                        'service:base': 'account', 'contactable': 'account'}}),
-                    ('econ:pay:instrument', {'template': {'instrument': 'account'}}),
+                    ('econ:pay:instrument', {}),
```

Comment:
'instrument' is still the name in the `econ:pay:instrument` template, not sure if that one was intended to be switched to title as well, there is still some 'instrument' usage around